### PR TITLE
Add basic vendoring scripts

### PR DESCRIPTION
**Pull request by sanket1729**
Copied and updated basic vendor scripts from the rust-secp256k1 repos. This does not no any symbol renaming, but is still handy and maintains which versions of simplicity we are using.

### CODE REVIEW

File: simplicity-sys/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+## Vendoring
+
+The default build process is to build using the vendored libsecp256k1 sources in
+the depend folder. There is no support for symbols yet.
+
+To update the vendored sources, use the `vendor-simplicity.sh` script:
+
+```
+$ ./vendor-libsecp.sh depend <rev>
```
Comment (apoelstra) [inline simplicity-sys/README.md:16]: In 27cbcfb96e0bc48dec0951661b93695f80ef1531:

`vendor-libsecp` should be `vendor_simplicity.sh`. (Also, consider renaming vendor_simplicity to vendor-simplicity.)

File: simplicity-sys/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+This crate provides Rust definitions for the FFI structures and methods.
+
+
+## Vendoring
+
+The default build process is to build using the vendored libsecp256k1 sources in
+the depend folder. There is no support for symbols yet.
+
+To update the vendored sources, use the `vendor-simplicity.sh` script:
```
Comment (apoelstra) [inline simplicity-sys/README.md:13]: In 27cbcfb96e0bc48dec0951661b93695f80ef1531:

This message uses `-` but the actual file uses `_`. I recommend renaming the file.

**Comment by apoelstra** [comment]
Other than the filename stuff this looks great! It's great that we can just run a revendoring script to check that simplicity updates were done correctly.

**Review by apoelstra — APPROVED** [review]
ACK 9f553818bd875aab44c420b51594c1b4a39195f2

**Comment by sanket1729** [comment]
Merging this and fixing these things in first commit of PR2.

**Pull request merged**